### ISSUE TITLE
No Liqudsoap in installation?

### ISSUE THREAD

**Issue by NicoxDJ** [AzuraCast/ansible-installer#2]
Hey! I've notice that there isn't liqudsoap in the installation...
The Ansible installation is broken. AzuraCast can't work.
Please advise

**Comment by btoplak** [comment]
Yes, it's there: `ansible/roles/azuracast-radio/tasks/liquidsoap.yml`
